lib/main.js

```diff
@@ -43,6 +49,17 @@ var runSass = function(sassOptions, trueOptions) {
   } else {
     sassOpts.includePaths = [sassPath];
   }
+
+  if (sassOpts.file) {
+    sassOpts.data = fs.readFileSync(sassOpts.file, 'utf-8');
+    sassOpts.includePaths.push(path.dirname(sassOpts.file));
+    delete sassOpts.file;
+  }
```

Comment:
This approach no longer works if the test is in an imported file, not the parent file being passed in. I'm not sure how to get around that without re-implementing the Sass import logic?

Reply:
This is related to your comment above regarding assertions nested within @imported files. I will test this in my attempts to fix the greater problem of importing test files.